pywt/tests/test_wavelet.py

```diff
@@ -264,3 +264,14 @@ def test_wavefun_bior13():
     assert_allclose(phi_r, phi_r_expect, rtol=1e-10, atol=1e-12)
     assert_allclose(psi_d, psi_d_expect, rtol=1e-10, atol=1e-12)
     assert_allclose(psi_r, psi_r_expect, rtol=1e-10, atol=1e-12)
+
+
+def test_wavelet_pickle(tmpdir):
+    wavelet = pywt.Wavelet('sym4')
+    filename = os.path.join(tmpdir, 'wav.pickle')
+    with open(filename, 'wb') as f:
+        pickle.dump(wavelet, f)
```

Comment:
This will leave the file lying around on disk after running the test. Rather than cleaning that up in a `try`-`finally`, perhaps use `dumps` instead?

Reply:
After testing: it actually doesn't, this is a pytest fixture: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture